cartographer_grpc/local_trajectory_uploader.h

```diff
@@ -56,60 +52,9 @@ class LocalTrajectoryUploaderInterface {
       int local_trajectory_id) const = 0;
 };
 
-class LocalTrajectoryUploader : public LocalTrajectoryUploaderInterface {
- public:
-  LocalTrajectoryUploader(const std::string& uplink_server_address);
-  ~LocalTrajectoryUploader();
-
-  // Starts the upload thread.
-  void Start();
-
-  // Shuts down the upload thread. This method blocks until the shutdown is
-  // complete.
-  void Shutdown();
-
-  void AddTrajectory(
-      int local_trajectory_id, const std::set<SensorId>& expected_sensor_ids,
-      const cartographer::mapping::proto::TrajectoryBuilderOptions&
-          trajectory_options) override;
-  void FinishTrajectory(int local_trajectory_id) override;
-  void EnqueueDataRequest(
-      std::unique_ptr<google::protobuf::Message> data_request) override;
-
-  SensorId GetLocalSlamResultSensorId(int local_trajectory_id) const override {
-    return SensorId{SensorId::SensorType::LOCAL_SLAM_RESULT,
-                    "local_slam_result_" + std::to_string(local_trajectory_id)};
-  }
-
- private:
-  void ProcessSendQueue();
-  void TranslateTrajectoryId(proto::SensorMetadata* sensor_metadata);
-  void ProcessFixedFramePoseDataMessage(
-      proto::AddFixedFramePoseDataRequest* data_request);
-  void ProcessImuDataMessage(proto::AddImuDataRequest* data_request);
-  void ProcessLocalSlamResultDataMessage(
-      proto::AddLocalSlamResultDataRequest* data_request);
-  void ProcessOdometryDataMessage(proto::AddOdometryDataRequest* data_request);
-  void ProcessLandmarkDataMessage(proto::AddLandmarkDataRequest* data_request);
-
-  std::shared_ptr<grpc::Channel> client_channel_;
-  std::map<int, int> local_to_cloud_trajectory_id_map_;
-  cartographer::common::BlockingQueue<
-      std::unique_ptr<google::protobuf::Message>>
-      send_queue_;
-  bool shutting_down_ = false;
-  std::unique_ptr<std::thread> upload_thread_;
-  std::unique_ptr<framework::Client<handlers::AddFixedFramePoseDataHandler>>
-      add_fixed_frame_pose_client_;
-  std::unique_ptr<framework::Client<handlers::AddImuDataHandler>>
-      add_imu_client_;
-  std::unique_ptr<framework::Client<handlers::AddLocalSlamResultDataHandler>>
-      add_local_slam_result_client_;
-  std::unique_ptr<framework::Client<handlers::AddOdometryDataHandler>>
-      add_odometry_client_;
-  std::unique_ptr<framework::Client<handlers::AddLandmarkDataHandler>>
-      add_landmark_client_;
-};
+// Returns LocalTrajectoryUploader with the actual implementation.
```

Comment:
Describe this change in the commit message.

Reply:
Done